imports/plugins/included/jobcontrol/server/jobs/cart.js

```diff
@@ -0,0 +1,91 @@
+import later from "later";
+import moment from "moment";
+import { Accounts, Cart, Jobs } from "/lib/collections";
+import { Hooks, Logger, Reaction } from "/server/api";
+
+const olderThan = moment().subtract(3, "days")._d;
+
+Hooks.Events.add("onJobServerStart", () => {
+  Logger.debug("Adding Job removeStaleCart to jobControl");
+  const staleCarts = Cart.find({
+    updatedAt: {
+      $lte: olderThan
+    }
+  }).fetch();
+  if (staleCarts) {
+    new Job(Jobs, "cart/removeFromCart", {})
+      .priority("normal")
+      .retry({
+        retries: 5,
+        wait: 60000,
+        backoff: "exponential" // delay by twice as long for each subsequent retry
+      })
+      .repeat({
+        schedule: later.parse.text("every day")
+      })
+      .save({
+        cancelRepeats: true
+      });
+  } else {
+    Logger.info("No stale carts at the moment");
```

Comment:
This shouldn't be an `info` level log. We don't need to log things we **don't** do.